content/docs/command-reference/import-url.md

```diff
@@ -31,7 +31,7 @@ external data source changes. Example scenarios:
 - A shared dataset on a remote storage that is managed and updated outside DVC.
 
 > Note that `dvc get-url` corresponds to the first step this command performs
-> (just download the file or directory).
+> (just downloads the file or directory).
```

Comment:
Good. But what about the other change from https://github.com/iterative/dvc.org/pull/1396/files#diff-06ee43874ed57c5e9a7512621fba5ed1L81-R81 ?

`` `remote://myremote/path/to/file` notation just means`` -> ``The `remote://myremote/path/to/file` notation means`` please